lm_eval/models/huggingface.py

```diff
@@ -358,6 +318,84 @@ def __init__(
                 f"Loglikelihood prefix token id used in evaluation: {self.prefix_token_id}"
             )
 
+    def _get_accelerate_args(
+        self,
+        parallelize: bool = None,
+        device_map: Optional[str] = "auto",
+        max_memory_per_gpu: Optional[Union[int, str]] = None,
+        max_cpu_memory: Optional[Union[int, str]] = None,
+        offload_folder: Optional[str] = "./offload",
+        gpus: Optional[int] = None,
+    ) -> dict:
+        """Returns the kwargs needed to apply `accelerate` in `AutoModel.from_pretrained`."""
+        num_local_processes = int(os.environ.get("LOCAL_WORLD_SIZE", 1))
```

Comment:
@NathanHB @clefourrier maybe the point of confusion is that I'm reading too much into the 
`num_machines` variable name--I assumed this indicated that this is handling the case where we have several nodes that we are splitting across.

That seems not to be the case--let's call this variable `num_machines` -> `num_dp_groups` instead or similar to reduce confusion?

Would there also be a quick way supported by Accelerate to catch and error explicitly if we have > 1 machine in use? Will investigate this myself

Reply:
I had a quick look around and didn't find something for Accelerate, but added a suggestion for explicitly telling users to be wary of this

Reply:
This should also allow to run jobs in multi node, technically - we could provide slurm scripts for this.
But it's not the core of the PR ^^

Reply:
Checks out! Agree it's not the core of the PR, I mentioned it's not explicitly supported in the README.

(do you all ever run multinode? curious if this is something that's being used actively.)

Reply:
We actually do, for all large scale evals

Reply:
But we've had issues recently for some models, so we're making sure we ported all the needed code